README.md

```diff
@@ -133,6 +133,93 @@ Additional options:
+| [tinyint][sql-all-int-ref]              | [Boolean][java-boolean-ref], [Byte*][java-byte-ref], [Short][java-short-ref], [Integer][java-integer-ref], [Long][java-long-ref] |
+| [smallint][sql-all-int-ref]             | [Boolean][java-boolean-ref], [Byte][java-byte-ref], [Short*][java-short-ref], [Integer][java-integer-ref], [Long][java-long-ref] |
+| [int][sql-all-int-ref]                  | [Boolean][java-boolean-ref], [Byte][java-byte-ref], [Short][java-short-ref], [Integer*][java-integer-ref], [Long][java-long-ref] |
+| [bigint][sql-all-int-ref]               | [Boolean][java-boolean-ref], [Byte][java-byte-ref], [Short][java-short-ref], [Integer][java-integer-ref], [Long*][java-long-ref] |
+| [float][sql-float-real-ref]             | [Double*][java-double-ref], [Float][java-float-ref] 
+| [real][sql-float-real-ref]              | [Double][java-double-ref], [Float*][java-float-ref]   
+| [decimal][sql-decimal-ref]              | [BigDecimal][java-bigdecimal-ref] 
+| [numeric][sql-decimal-ref]              | [BigDecimal][java-bigdecimal-ref]
+| [uniqueidentifier][sql-uid-ref]         | [UUID*][java-uuid-ref], [String][java-string-ref]   
+| [smalldatetime][sql-smalldatetime-ref]  | [LocalDateTime][java-ldt-ref] 
+| [datetime][sql-datetime-ref]            | [LocalDateTime][java-ldt-ref] 
+| [datetime2][sql-datetime2-ref]          | [LocalDateTime][java-ldt-ref] 
+| [date][sql-date-ref]                    | [LocalDate][java-ld-ref] 
+| [time][sql-time-ref]                    | [LocalTime][java-time-ref] 
+| [timestamp][sql-timestamp-ref]          | byte[]
+| [datetimeoffset][sql-dtof-ref]          | [OffsetDateTime][java-odt-ref], [ZonedDateTime][java-zdt-ref]  
+| [smallmoney][sql-money-ref]             | [BigDecimal][java-bigdecimal-ref]
+| [money][sql-money-ref]                  | [BigDecimal][java-bigdecimal-ref]
+| [char][sql-(var)char-ref]               | [String][java-string-ref]
+| [varchar][sql-(var)char-ref]            | [String][java-string-ref]
+| [varcharmax][sql-(var)char-ref]         | ---
+| [nchar][sql-n(var)char-ref]             | [String][java-string-ref]
+| [nvarchar][sql-n(var)char-ref]          | [String][java-string-ref]
+| [nvarcharmax][sql-n(var)char-ref]       | ---
+| [text][sql-(n)text-ref]                 | ---
+| [ntext][sql-(n)text-ref]                | ---
+| [image][sql-(n)text-ref]                | ---
+| [binary][sql-binary-ref]                | ---
+| [varbinary][sql-binary-ref]             | ---
+| [varbinarymax][sql-binary-ref]          | ---
+| [sql_variant][sql-sql-variant-ref]      | ---
+| [xml][sql-xml-ref]                      | ---
+| [udt][sql-udt-ref]                      | ---
+| [geometry][sql-geometry-ref]            | ---
+| [geography][sql-geography-ref]          | ---
+
+_**\* Indicates preferred java types while decoding.**_
```

Comment:
Let's indicate the default type by using the order in which types are mentioned.

Reply:
I put them first and making also bold